src/da/activate.ts

```diff
@@ -13,13 +14,22 @@ export function activateDebugger(context: vscode.ExtensionContext) {
       if (!targetResource && vscode.window.activeTextEditor) {
         targetResource = vscode.window.activeTextEditor.document.uri;
       }
+
+      // Only add the inputPath if the file exists
+      var inputPath = getInputPath();
+      if (!existsSync(inputPath)) {
+        inputPath = "";
```

Comment:
Is this handled as no input path?

Reply:
Yes, but I'll change this to be `undefined` if no file is found instead, as that feels a bit more idiomatic.